lib/Auth/Utils.php

```diff
@@ -117,7 +117,7 @@ public static function authenticatePI(array &$state, array $formParams): ?PIResp
             {
                 try
                 {
-                    $response = $pi->validateCheckWebAuthn($username, $transactionID, $webAuthnSignResponse, $origin);
+                    $response = $pi->validateCheckWebAuthn($username, $transactionID, $webAuthnSignResponse, $origin, $headersToForward);
```

Comment:
rename $headersToForward to $headers

Reply:
fixed [f2bb187](https://github.com/privacyidea/simplesamlphp-module-privacyidea/pull/198/commits/f2bb187f584e1a7a8c5c5fec38dee8d19a3e6b78)